libs/pika/async_combinators/include/pika/async_combinators/when_all.hpp

```diff
@@ -130,25 +130,25 @@ namespace pika {
 
 #else    // DOXYGEN
 
-#include <pika/config.hpp>
-#include <pika/allocator_support/internal_allocator.hpp>
-#include <pika/futures/detail/future_data.hpp>
-#include <pika/futures/detail/future_transforms.hpp>
-#include <pika/futures/future.hpp>
-#include <pika/futures/traits/acquire_future.hpp>
-#include <pika/futures/traits/acquire_shared_state.hpp>
-#include <pika/futures/traits/future_access.hpp>
-#include <pika/futures/traits/future_traits.hpp>
-#include <pika/futures/traits/is_future.hpp>
-#include <pika/futures/traits/is_future_range.hpp>
-#include <pika/pack_traversal/pack_traversal_async.hpp>
+# include <pika/config.hpp>
+# include <pika/allocator_support/internal_allocator.hpp>
+# include <pika/futures/detail/future_data.hpp>
+# include <pika/futures/detail/future_transforms.hpp>
+# include <pika/futures/future.hpp>
```

Comment:
These are the only "bad" consequences of `IndentPPDirectives` in my opinion. It's not bad enough for me to say no to this, since I think it improves other places, but I just wanted to point this out in case you didn't notice. @aurianer, @biddisco are you fine with this type of changes?